.github/workflows/ci.yml

```diff
@@ -14,6 +14,25 @@ env:
   ICUB_TAG: v2.8.2
 
 jobs:
+  format-check:
+    name: '[format-check]'
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Check formatting
+      run: |
+        git diff --exit-code
+        git diff --exit-code --cached
+
+    - name: Install Clang Format
+      run: sudo apt-get install clang-format
+
+    - name: Run Clang Format
+      run: |
+        find . -type f \( -name '*.cpp' -o -name '*.h' -o -name '*.hpp' \) -exec clang-format -i {} \;
```

Comment:
What is this doing? Why it is running on `ubuntu-20.04` ? Can we do it with pixi to avoid duplicating logic?